libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/item/event/TimelineEventContentMapper.kt

```diff
@@ -96,6 +96,7 @@ class TimelineEventContentMapper(private val eventMessageMapper: EventMessageMap
                     data = kind.msg.map()
                 )
             }
+            else -> UnknownContent
```

Comment:
Maybe we should add the 2 poll contents here explicitly with a TODO/FIXME?

Reply:
I guess @julioromano has already done it locally...